ampersand-form-view.js

```diff
@@ -183,7 +187,7 @@ module.exports = View.extend({
             delete this._startingValues;
         }
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.el.addEventListener('submit', this.handleSubmit, false);
+        this.formEl.addEventListener('submit', this.handleSubmit, false);
```

Comment:
i have bias to retaining `el` vs `formEl`.  programmatically when i reference my form in my app it reads kind of funny to me to do `myForm.formEl` vs `myForm.el`.  pedantic, surely, but also a major API change that I'm not sure is warranted